### ISSUE TITLE
Deploy in Elastic Beanstalk AWS

### ISSUE THREAD

**Issue by ns-ffx** [ragapp/ragapp#67]
Are there any requirements for deploying it in Elastic Beanstalk? Do I need to change the docker-compose.yml file?
For now it seems that I can't reach my services when hosting there.

EDIT: Ok Solved with deployment in EC2 in aws.

**Comment by marcusschiesser** [comment]
@ns-ffx which docker-compose.yml file did you use? want to share?

**Comment by ns-ffx** [comment]
I Marcus. I just used the same docker image you provided with the same docker-compose file for EC2. works perfectly fine now! What I am right now figuring out is how to configure the ragapp to be able to upload, delete & get file list from a different domain. I guess I need to update the CORS policy.

**Comment by marcusschiesser** [comment]
@ns-ffx do you mind sharing with us how you did the deployment? would be helpful also for other users

**Comment by ns-ffx** [comment]
@marcusschiesser Yes, I used AWS EC2: 
1) Amazon Linux 2 Configuration. 
2) I allowed HTTP & HTTPS traffic in the config setup.
on the instance:
3) yum update (on the instance)
4) sudo amazon-linux-extras install docker
5) sudo service docker start
5) sudo usermod -a -G docker ec2-user (practical)
6) sudo docker image pull ragapp/ragapp
7) sudo docker run -d -p 80:8000 ragapp/ragapp

That did it for me for the first prototype running on the instance. Hope that helps.
